### ISSUE TITLE
#div:nth-child(1) > .gWxTCi.StyledListItem-sc-1sb8owe-1[draggable="true"] > .StyledListTextContainer-sc-1sb8owe-5.bgEsee > .StyledListSubtitle-sc-1sb8owe-4.jhpUkl.sc-dmRaPn > .sc-kgflAQ.bAlVxb:nth-child(2)------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by Khyati007** [khyatit13/Coding-practice#314]
Source: `<div class="sc-kgflAQ bAlVxb">Products, Survey123</div>`
Element Location: div:nth-child(1) > .gWxTCi.StyledListItem-sc-1sb8owe-1[draggable="true"] > .StyledListTextContainer-sc-1sb8owe-5.bgEsee > .StyledListSubtitle-sc-1sb8owe-4.jhpUkl.sc-dmRaPn > .sc-kgflAQ.bAlVxb:nth-child(2)
failureSummary: Fix any of the following:
  Element has insufficient color contrast of 3.03 (foreground color: #949494, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1Tags: [Ljava.lang.String;@4a067c25
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
